### PR TITLE
[Docs] Fix merge errors / start vale checking in docs 

### PR DESCRIPTION
**Pull request by stichbury**
## Description
Now ready for review. Sorry it's so large, it's because I have touched most of the docs pages at a superficial level to remove a few common issues that Vale is set to eliminate:

* Remove e.g. and replace with "for example"
* Remove "please" as unnecessarily wordy
* Remove "simple" and "just" as potentially making the user feel bad
* Replace some commonly used complex terms like "provide" with "give" and so on
* Add some line spacing to stop code sections from being dragged into Vale check
* Fix some link issues (break the build rather than Vale)

This PR also turns Vale on to run across all markdown in the `/docs` folders (thanks to @huong-li-nguyen).

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

**Comment by huong-li-nguyen** [comment]
I've just pushed a change 👍  From some of the threads I've quickly read e.g. [this one](https://github.com/errata-ai/vale/issues/131), it seems like `vale` works the other way around. Instead of ignoring folders, you have to specify which ones you want to lint.

To test it, I just added "werkzeug" in the vizro-core or vizro-ai `pages/docs/*.md` files and then checked if vale was raising it and it did, but please double-check.

File: vizro-core/docs/pages/user-guides/actions.md [hunk 1]
```diff
@@ -170,7 +173,7 @@ Here is an example of how to configure a chart interaction when the source is a
         ```
     === "app.yaml"
         ```yaml
-        # Still requires a .py to add data to the data manager and parse YAML configuration
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration
```
Comment (huong-li-nguyen) [inline vizro-core/docs/pages/user-guides/actions.md:176]: I don't remember what Antony said again, but as far as I remember we shouldn't use data connector, data set etc. but just data? Could you double-check in his PR on the data manager 🤔

Comment (stichbury) [reply vizro-core/docs/pages/user-guides/actions.md]: Thanks, this was an issue in the merge which I thought I'd fixed. I've grepped all the source for data connector/dataset and done a line by line comparison of Antony's changes and mine, and I think/hope/pray we are all good now 🙏

File: vizro-core/docs/pages/tutorials/first-dashboard.md [hunk 2]
```diff
@@ -81,7 +84,7 @@ Paste the following example into a Notebook cell, run it, and view the generated
 
     [Dashboard]: ../../assets/user_guides/dashboard/dashboard.png
 
-
+<!-- vale off -->
 ### 4. Explore further
-
-You are now ready to explore Vizro further, by working through the [Explore Vizro tutorial](../tutorials/explore-components.md) or by consulting the [how-to guides](../user-guides/dashboard.md).
+<!-- vale on -->
+You are now ready to explore Vizro further, by working through the [Explore Vizro tutorial](./explore-components.md) or by consulting the [how-to guides](../user-guides/dashboard.md).
```
Comment (huong-li-nguyen) [inline vizro-core/docs/pages/tutorials/first-dashboard.md:90]: Shall we add "" or similar for page titles?

```suggestion
You are now ready to explore Vizro further, by working through the ["Explore Vizro" tutorial](./explore-components.md) or by consulting the [how-to guides](../user-guides/dashboard.md).
```

File: vizro-core/docs/pages/user-guides/actions.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 # How to use actions
 
-This guide shows you how to use actions, a concept that is similar, but not identical, to [callbacks](https://dash.plotly.com/basic-callbacks) in `Dash`.
-Many components of a dashboard (e.g. [`Graph`][vizro.models.Graph] or [`Button`][vizro.models.Button]) have an optional `actions` argument, where you can enter the [`Action`][vizro.models.Action] model.
+This guide shows you how to use actions, an idea that is similar to [callbacks](https://dash.plotly.com/basic-callbacks) in `Dash`.
+Many components of a dashboard (for example, [`Graph`][vizro.models.Graph] or [`Button`][vizro.models.Button]) have an optional `actions` argument, where you can enter the [`Action`][vizro.models.Action] model.
 
-In a nutshell, using the [`Action`][vizro.models.Action] model together with an action function allows you to create complex functionality on a variety of triggers in your dashboard.
+In a nutshell, using the [`Action`][vizro.models.Action] model together with an action function enables you to create complex functionality on a variety of triggers in your dashboard.
 There is already a range of reusable action functions available.
```
Comment (huong-li-nguyen) [inline vizro-core/docs/pages/user-guides/actions.md:7]: Would shorten a bit?

```suggestion
By combining the [`Action`][vizro.models.Action] model with an action function, you can create complex dashboard interactions triggered by various events. 

There are already lots of action functions you can reuse.
```

File: vizro-core/docs/pages/user-guides/assets.md [hunk 4]
```diff
@@ -97,9 +99,9 @@ following the pattern `"{component_id}_outer"`.
 
 To achieve this, do the following:
 
-1. Provide a custom `id` to the relevant `Card` e.g `Card(id="my_card", ...)`
-2. Take a look at the source code of the component to see which CSS Class you need to target e.g. `"card"` or `"card_text"`
-3. Use CSS selectors to target the right property e.g. by leveraging the ID of the outermost Div `"my_card_outer"`
+1. Pass a custom `id` to the relevant `Card` e.g `Card(id="my_card", ...)`
+2. Take a look at the source code of the component to see which CSS Class you need to target such as `"card"` or `"card_text"`
```
Comment (huong-li-nguyen) [inline vizro-core/docs/pages/user-guides/assets.md:103]: card-text doesn't exist anymore

```suggestion
2. Take a look at the source code of the component to see which CSS Class you need to target such as `"card"`
```

File: vizro-core/docs/pages/user-guides/card-button.md [hunk 5]
```diff
@@ -404,12 +405,12 @@ and provide an attribute selector to select images with that matching URL hash.
...
 
-A navigation card allows you to navigate to a different page via a click on the card area.
+A navigation card enables you to navigate to a different page via a click on the card area.
 To create a navigation card, do the following:
 
 - Insert the [`Card`][vizro.models.Card] into the `components` argument of the [`Page`][vizro.models.Page]
-- Provide the `text` parameter with a title and some description
-- Provide the `href` parameter (relative or absolute URL)
+- Give the `text` parameter with a title and some description
+- Give the `href` parameter (relative or absolute URL)
```
Comment (huong-li-nguyen) [inline vizro-core/docs/pages/user-guides/card-button.md:413]: **Provide** sounds better here - doesn't it?

"Give the `text` parameter with a title and some description" sounds a bit odd to me 😄

File: vizro-core/docs/pages/user-guides/custom-actions.md [hunk 6]
```diff
@@ -134,7 +137,7 @@ The following example shows how to create a custom action that shows the clicked
         Vizro().build(dashboard).run()
         ```
 
-        1. Just as for a normal Python function, the names of the arguments `show_species` and `points_data` are arbitrary and do not need to match on to the names of `inputs` in any particular way.
+        1. Just as for a standard Python function, the names of the arguments `show_species` and `points_data` are arbitrary and do not need to match on to the names of `inputs` in any particular way.
```
Comment (huong-li-nguyen) [inline vizro-core/docs/pages/user-guides/custom-actions.md:140]: Not sure what a normal or standard Python function is 😄  I think we mean any Python function right?

```suggestion
        1. Just as for any Python function, the names of the arguments `show_species` and `points_data` are arbitrary and do not need to match on to the names of `inputs` in any particular way.
```

File: vizro-core/docs/pages/user-guides/filters.md [hunk 7]
```diff
@@ -64,7 +64,7 @@ automatically based on the target column, e.g. a dropdown for categorical data,
 
 ## Changing selectors
 
-If you want to have a different selector for your filter, you can provide the `selector` argument of the [`Filter`][vizro.models.Filter] with a different selector model.
+If you want to have a different selector for your filter, you can pass the `selector` argument of the [`Filter`][vizro.models.Filter] with a different selector model.
```
Comment (huong-li-nguyen) [inline vizro-core/docs/pages/user-guides/filters.md:67]: I saw this replacement a couple of times now and it always sounds a bit odd to me, but I'm no native speaker 😄 Isn't it **pass to** something instead of *pass with*. Everytime `provide` was replaced with `pass` it sounds a bit odd?

File: vizro-core/docs/pages/user-guides/pages.md [hunk 8]
```diff
@@ -21,11 +21,11 @@ A [`Page`][vizro.models.Page] is split up into four main containers:
 
 To create and add a page to your dashboard, do the following steps:
 
-1. Provide a `title` to your [`Page`][vizro.models.Page]
+1. Set a `title` for your [`Page`][vizro.models.Page]
 2. Configure your `components`, see our guide on the [various options](components.md)
 3. (optional) Configure your `controls` , see our guides on [Filters](filters.md) and [Parameters](parameters.md)
 4. (optional) Configure your `layout` , see our guide on [Layouts](layouts.md)
-5. (optional) Provide a `description` of your `Page` for the app's [meta tags](https://metatags.io/)
+5. (optional) Give a `description` of your `Page` for the app's [meta tags](https://metatags.io/)
```
Comment (huong-li-nguyen) [inline vizro-core/docs/pages/user-guides/pages.md:28]: Is it **give for** or **give to**?

File: vizro-core/docs/pages/user-guides/run.md [hunk 9]
```diff
@@ -24,9 +24,10 @@ This guide shows you how to launch your dashboard in different ways. By default,
 
         Vizro().build(dashboard).run()
         ```
-- create a python file named app.py.
+- create a Python file named `app.py`.
 - type the command `python app.py` into your terminal.
-- information below will be displayed in your terminal, go to the http link.
+- information below will be displayed in your terminal, go to the [http://127.0.0.1:8050/](http://127.0.0.1:8050/).
```
Comment (huong-li-nguyen) [inline vizro-core/docs/pages/user-guides/run.md:29]:
```suggestion
- information below will be displayed in your terminal, go to [http://127.0.0.1:8050/](http://127.0.0.1:8050/).
```

File: vizro-core/docs/pages/user-guides/selectors.md [hunk 10]
```diff
@@ -143,4 +143,4 @@ indicating that [pandas.api.types.is_datetime64_any_dtype()](https://pandas.pyda
     [Filter]: ../../assets/user_guides/selectors/default_filter_selectors.png
 
 
-To enhance existing selectors, please see our How-to-guide on creating [custom components](custom-components.md).
+To enhance existing selectors, see our how-to-guide on creating [custom components](custom-components.md).
```
Comment (huong-li-nguyen) [inline vizro-core/docs/pages/user-guides/selectors.md:146]: I'm trying to find this link Antony once sent to me, but I can't find it quickly ..
But basically it was a link to the best practices of Kedro docs, so you might be familiar with it @stichbury .

It provided some guidance that one should not use "custom components" as a hyperlink but "creating custom components", so always providing context of what the hyperlink contains instead of only having "custom components" be the hyperlink. Not sure if I make any sense 😄

**Review by huong-li-nguyen — COMMENTED** [review]
Hey @stichbury - LGTM in general! Thanks for tidying 🧹 

I've noticed a couple other merge conflicts which I pointed out. Some of the rephrases sound a bit odd to me now, but it might just be because I am not a native speaker 😄 Especially the replacements from "provide" to "pass" sound a bit off, because it now says "pass with ..." and "pass to" sounds more correct to me? Anyways, I've commented on the relevant sections. So if relevant please take a look, if not, then feel free to ignore :)

File: vizro-core/docs/pages/user-guides/filters.md (hunk 7 above)
Comment (huong-li-nguyen) [reply vizro-core/docs/pages/user-guides/filters.md:67]: And here I think it should be the other way around if we use "pass". You can pass a different selector model to the `selector` argument of the Filter. The current phrasing confuses me a lot

File: vizro-core/docs/pages/user-guides/selectors.md (hunk 10 above)
Comment (huong-li-nguyen) [reply vizro-core/docs/pages/user-guides/selectors.md:146]: Here it is: https://github.com/kedro-org/kedro/wiki/Kedro-documentation-style-guide#links

```
Links
Make hyperlink descriptions as descriptive as you can. This is a good description:

Learn how to [update the project pipeline](https://docs.kedro.org/en/stable/tutorial/create_pipelines.html#update-the-project-pipeline)

This is less helpful:

Learn how to update the [project pipeline](https://docs.kedro.org/en/stable/tutorial/create_pipelines.html#update-the-project-pipeline)
```
Don't write this:

To learn how to update the project pipeline, see [here](https://docs.kedro.org/en/stable/tutorial/create_pipelines.html#update-the-project-pipeline)

Comment (stichbury) [reply vizro-core/docs/pages/user-guides/selectors.md:146]: You're right! This set of fixes is just to get Vale to pass rather than to make all the changes necessary to perfect the docs, so I've not tackled everything like this in this PR (although it sure does look like it from the number of changes!). It would be great to have Vale check hyperlinks for content -- I'm not sure it can do that but I have asked on their discussion forum https://github.com/errata-ai/vale/discussions/807

I have ticketed it (on this repo, it's a nice external issue for a potential contributor) https://github.com/mckinsey/vizro/issues/424

Comment (stichbury) [inline vizro-core/docs/pages/user-guides/selectors.md:146]:
```suggestion
To enhance existing selectors, see our [how-to-guide on creating custom components](custom-components.md).
```

File: vizro-core/docs/pages/development/contributing.md [hunk 11]
```diff
@@ -1,36 +1,36 @@
...
 
-    If you haven't used Hatch before, it's well worth skimming through [their documentation](https://hatch.pypa.io/), in particular the page on [environments](https://hatch.pypa.io/latest/environment/). Run `hatch env show` to show all of Hatch's environments and available scripts, and take a look at [`hatch.toml`](https://github.com/mckinsey/vizro/tree/main/vizro-core/hatch.toml) to see our Hatch configuration. It is useful handy to [Hatch's tab completion](https://hatch.pypa.io/latest/cli/about/#tab-completion) to explore the Hatch CLI.
+    If you haven't used Hatch before, it's well worth skimming through [their documentation](https://hatch.pypa.io/), in particular the page on [environments](https://hatch.pypa.io/latest/environment/). Run `hatch env show` to show all Hatch's environments and available scripts, and take a look at [`hatch.toml`](https://github.com/mckinsey/vizro/tree/main/vizro-core/hatch.toml) to see our Hatch configuration. It is useful handy to [Hatch's tab completion](https://hatch.pypa.io/latest/cli/about/#tab-completion) to explore the Hatch CLI.
```
[the remark this answers is not in the document]
Comment (stichbury) [reply vizro-core/docs/pages/development/contributing.md]: Ha ha! I've no idea -- Vale didn't flag it so I didn't change that bit of the text -- but I've fixed it now :)

File: vizro-core/docs/pages/tutorials/first-dashboard.md (hunk 2 above)
Comment (stichbury) [reply vizro-core/docs/pages/tutorials/first-dashboard.md]: This is such a difficult one to decide as the links aren't very consistent -- one's a title the other is a group of pages. Let's try your idea!

File: vizro-core/docs/pages/user-guides/card-button.md (hunk 5 above)
Comment (stichbury) [inline vizro-core/docs/pages/user-guides/card-button.md:413]:
```suggestion
- The `text` parameter contains a title and some description.
- The `href` parameter is for a relative or absolute URL.
```

File: vizro-core/docs/pages/user-guides/filters.md (hunk 7 above)
Comment (stichbury) [inline vizro-core/docs/pages/user-guides/filters.md:67]:
```suggestion
If you want to have a different selector for your filter, you can give the `selector` argument of the [`Filter`][vizro.models.Filter] a different selector model.
```

Comment (stichbury) [reply vizro-core/docs/pages/user-guides/filters.md:67]: @huong-li-nguyen Does this work better? OR 

```suggestion
If you want to have a different selector for your filter, you can pass a different select model as the `selector` argument of the [`Filter`][vizro.models.Filter].
```

File: vizro-core/docs/pages/user-guides/pages.md (hunk 8 above)
Comment (stichbury) [reply vizro-core/docs/pages/user-guides/pages.md:28]:
```suggestion
5. (optional) Give a `description` of your `Page` to the app's [meta tags](https://metatags.io/)
```

File: vizro-core/docs/pages/API-reference/manager.md [hunk 12]
```diff
@@ -0,0 +1,6 @@
+# Data Manager
+<!-- vale off -->
+
+::: vizro.managers._data_manager
+
+<!-- vale on -->
```
[the remark this answers is not in the document]
Comment (stichbury) [reply vizro-core/docs/pages/API-reference/manager.md:6]: Removed

File: vizro-core/docs/pages/user-guides/filters.md (hunk 7 above)
Comment (petar-qb) [inline vizro-core/docs/pages/user-guides/filters.md:67]: What do you think about this one?
```suggestion
If you want to have a different selector for your filter, you can define the `selector` argument of the [`Filter`][vizro.models.Filter] with a different selector model.
```

File: vizro-core/docs/pages/user-guides/selectors.md (hunk 10 above)
Comment (huong-li-nguyen) [reply vizro-core/docs/pages/user-guides/selectors.md]: Perfect, thanks!

File: vizro-core/docs/pages/user-guides/filters.md (hunk 7 above)
Comment (huong-li-nguyen) [reply vizro-core/docs/pages/user-guides/filters.md:67]: The first one is better! 👍 so

If you want to have a different selector for your filter, you can give the `selector` argument of the [`Filter`][vizro.models.Filter] a different selector model.

File: vizro-core/docs/pages/user-guides/card-button.md [hunk 13]
```diff
@@ -404,12 +405,12 @@ and provide an attribute selector to select images with that matching URL hash.
...
 
-A navigation card allows you to navigate to a different page via a click on the card area.
+A navigation card enables you to navigate to a different page via a click on the card area.
 To create a navigation card, do the following:
 
 - Insert the [`Card`][vizro.models.Card] into the `components` argument of the [`Page`][vizro.models.Page]
-- Provide the `text` parameter with a title and some description
-- Provide the `href` parameter (relative or absolute URL)
+- The `text` parameter contains a title and some description.
+- The `href` parameter is for a relative or absolute URL.
```
Comment (huong-li-nguyen) [inline vizro-core/docs/pages/user-guides/card-button.md:413]: I think this might have worked better before. It probably didn't become very clear but this should be instructions of how to create the navigation card so:

1. Do this
2. Then do this
3. Do this

It doesn't have to be in that order though. So I think it should start with a verb - WDYT @stichbury ?

Maybe:


```suggestion
- Pass your markdown text to the `Card.text`
- Pass a relative or absolute URL to the `Card.href`
```

**Review by huong-li-nguyen — APPROVED** [review]
LGTM 👍 Just have some minor comments, but from my side it's good to go 🚀

File: vizro-ai/docs/pages/explanation/safety-in-vizro-ai.md [hunk 14]
```diff
@@ -6,39 +6,44 @@ We recommend users research and understand the selected model before using `vizr
...
 <!-- vale off -->
+### 1. Hallucination and misrepresentation
+
 Generative models can potentially generate information while appearing factual, being entirely fictitious or misleading.
 <!-- vale on -->
 The vendor models might lack real-time knowledge or events beyond its last updates.
 `vizro_ai` output may vary and you should always verify critical information.
 It is the user's responsibility to discern the accuracy, consistent, and reliability of the generated content.
 
+<!-- vale off -->
```
Comment (maxschulz-COL) [inline vizro-ai/docs/pages/explanation/safety-in-vizro-ai.md:18]: Will this be turned on later again?

File: vizro-core/docs/pages/user-guides/custom-actions.md [hunk 15]
```diff
@@ -1,9 +1,10 @@
 # How to create custom actions
 
-This guide demonstrates the usage of custom actions, a concept that shares similarities with, but is not identical to [callbacks](https://dash.plotly.com/basic-callbacks) in `Dash`.
+This guide demonstrates the usage of custom actions, a idea that shares similarities with, but is not identical to [callbacks](https://dash.plotly.com/basic-callbacks) in `Dash`.
```
Comment (maxschulz-COL) [inline vizro-core/docs/pages/user-guides/custom-actions.md:3]: Is it natural to write `a idea`?

File: vizro-core/docs/pages/user-guides/pages.md [hunk 16]
```diff
@@ -21,11 +21,11 @@ A [`Page`][vizro.models.Page] is split up into four main containers:
 
 To create and add a page to your dashboard, do the following steps:
 
-1. Provide a `title` to your [`Page`][vizro.models.Page]
+1. Set a `title` for your [`Page`][vizro.models.Page]
 2. Configure your `components`, see our guide on the [various options](components.md)
 3. (optional) Configure your `controls` , see our guides on [Filters](filters.md) and [Parameters](parameters.md)
 4. (optional) Configure your `layout` , see our guide on [Layouts](layouts.md)
-5. (optional) Provide a `description` of your `Page` for the app's [meta tags](https://metatags.io/)
+5. (optional) Give a `description` of your `Page` to the app's [meta tags](https://metatags.io/)
```
Comment (maxschulz-COL) [inline vizro-core/docs/pages/user-guides/pages.md:28]: Is this correct now? You are not really giving it to the tags, but you are giving it so that the tags change. Maybe a detail...

File: vizro-core/src/vizro/managers/_data_manager.py [hunk 17]
```diff
@@ -63,11 +63,10 @@ def memoize(wrapped: BoundMethod, instance: _DynamicData, args, kwargs):
     Args:
         wrapped: bound method to memoize.
         instance: _DynamicData object to which wrapped is bound.
-        args: positional arguments supplied to wrapped, taken into account for generating cache key.
-        kwargs: keyword arguments supplied to wrapped, taken into account for generating cache key.
+        args : positional arguments supplied to wrapped, taken into account for generating cache key.
+        kwargs : keyword arguments supplied to wrapped, taken into account for generating cache key.
```
Comment (maxschulz-COL) [inline vizro-core/src/vizro/managers/_data_manager.py:67]: Is that according to the google style for docstrings

**Review by maxschulz-COL — APPROVED** [review]
Had a quick scan through - looks generally good 💪 

Main question: is there a guidance when we must turn `vale` off?

Some things I noticed:
- e.g. into for example -> is this not too long?

**Comment by petar-qb** [comment]
After searching the `vizro-core/docs` files and some keywords I found the following:
- some forms of `e.g.` in: `graph.md - 19`, `why-vizro.md - 100`, `why-vizro.md - 107`, `assets.md - 102`, `run. md - 48`.
- `etc` in:`cotributing.md - 111`, `why-vizro - 69`, `custom-components.md - several places`,
- some form of `i.e.` - Nowhere.

Also in `vizro-ai/docs` I found the following:
- `e.g.` in: `safeguard.md 75, 76`,  

I don't know if we should strictly get rid of them. However, here's a little warning in case it helps 😄.

File: vizro-core/src/vizro/managers/_data_manager.py (hunk 17 above)
Comment (stichbury) [reply vizro-core/src/vizro/managers/_data_manager.py:67]: @maxschulz-COL It's necessary because otherwise `griffe` reports a warning and, because of `--strict` that translates to an error. I don't know why @antonymilne didn't have this when he merged though 🤷

File: vizro-core/docs/pages/development/contributing.md [hunk 18]
```diff
@@ -161,17 +161,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+Begin by uncommenting the relevant section(s) you wish to describe. If your PR includes changes that are not relevant to `CHANGELOG.md`, leave everything commented out. If you are uncertain about what to add or whether to add anything, refer to [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). The rule of thumb should be, if in doubt, or if the user is affected in any way, it should be described in the `CHANGELOG.md`.
 
 ## Releases
 
-Vizro's version is given by `__version__` in [`src/vizro/__init__.py`](https://github.com/mckinsey/vizro/blob/main/vizro-core/src/vizro/__init__.py). To bump the version, run, e.g. `hatch version minor`. See [Hatch's documentation](https://hatch.pypa.io/latest/version/) for more details.
+Vizro's version is given by `__version__` in [`src/vizro/__init__.py`](https://github.com/mckinsey/vizro/blob/main/vizro-core/src/vizro/__init__.py). To bump the version, run `hatch version minor`. See [Hatch's documentation](https://hatch.pypa.io/latest/version/) for more details.
```
[the remark this answers is not in the document]
Comment (stichbury) [reply vizro-core/docs/pages/development/contributing.md:174]: Ticket: https://github.com/McK-Internal/vizro-internal/issues/738

File: vizro-ai/docs/pages/explanation/safety-in-vizro-ai.md (hunk 14 above)
Comment (stichbury) [reply vizro-ai/docs/pages/explanation/safety-in-vizro-ai.md:18]: @maxschulz-COL I'm turning it off in comments when the rule is generally a good one (e.g. to enforce correct capitalisation in headings) but when it fails for a valid reason and I can't work out how to work around it my editing the style. In this case, it doesn't like numbered headings, so I have excluded them from checking. My skills with regex mean I've not work out how to do the exception in the style!

File: vizro-core/src/vizro/managers/_data_manager.py (hunk 17 above)
Comment (maxschulz-COL) [reply vizro-core/src/vizro/managers/_data_manager.py:67]: Strange... given that we do not have automatic API docs for the data manager, I would then just leave as is and not fuss anymore!

**Comment by stichbury** [comment]
> Some things I noticed:
> 
> * e.g. into for example -> is this not too long?

Good question! In some cases, I removed the "e.g." and didn't replace with "for example" because it does make it wordy, but in others, it is useful to spell out that it's just one of many options/examples. We are familiar with `e.g.` but not everyone is, so technical writing tends to spell it out for clarity. It's one of those things that some like, some don't (and _someone_ in the team uses it a lot!), so I do apologise if you fall on the other side of the rule for this one!

File: vizro-core/src/vizro/managers/_data_manager.py (hunk 17 above)
Comment (stichbury) [reply vizro-core/src/vizro/managers/_data_manager.py:67]: That's the mystery solved! It was fussing because I accidentally re-included the page that pulls it in (@huong-li-nguyen spotted that my merge reintroduced it) so I fixed it, but it wasn't actually a problem for @antonymilne because it's been removed as you say. Thanks for clearing that up. Let's leave the fix in place for now though.

**Comment by stichbury** [comment]
> After searching the `vizro-core/docs` files and some keywords I found the following:
> 
> * some forms of `e.g.` in: `graph.md - 19`, `why-vizro.md - 100`, `why-vizro.md - 107`, `assets.md - 102`, `run.md - 48`.
> * `etc` in:`cotributing.md - 111`, `why-vizro - 69`, `custom-components.md - several places`,
> * some form of `i.e.` - Nowhere.
> 
> Also in `vizro-ai/docs` I found the following:
> 
> * `e.g.`and `etc` in: `safeguard.md 75, 76`,
> 


I don't need Vale with @petar-qb doing the checking :) Thanks for the spot!

Some may be excluded because I've turned Vale off on docs with official type text like the safeguarding page. I'll look into why others haven't been flagged and fix them all for consistency. Thanks @petar-qb !

File: vizro-core/docs/pages/development/contributing.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (stichbury) [reply vizro-core/docs/pages/development/contributing.md:174]: Ticketed https://github.com/McK-Internal/vizro-internal/issues/738
